### PR TITLE
finish

### PR DESCRIPTION
**Pull request by BovkunLeonid**

### CODE REVIEW

File: 32_communication_server/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <title>32 Communication width the server</title>
+    <script src="app.js" async></script>
+</head>
+<body>
+    <div class="container">
+        <h1 class="mb-3">Список справ</h1>
+        <div class="row mb-3">
+            <div class="col-auto">
+                <input id="todoText" type="text" class="form-control" placeholder="нова справа">
```
Comment (RomanIlkiv) [inline 32_communication_server/index.html:15]: don't use capital letter in name of html classes and ids

File: 32_communication_server/app.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                <input onchange="toggleCompleteTodo('${todo.id}')" type="checkbox" class="form-check-input" ${todo.completed && 'checked'}>
+                ${todo.title}
+            </label>
+            <button onclick="deleteTodo('${todo.id}')" type="button" class="btn-close" aria-label="Close"></button>
+        </div>
+    `);
+};
+
+// Create - створити; метод POST - створення данних (їх опублікування); _________________________________
+document.getElementById('addTodo').addEventListener('click', () => {
```
Comment (RomanIlkiv) [inline 32_communication_server/app.js:36]: camel case naming of variables is only valid for JS
in HTML and CSS we don't use capital cases

File: 32_communication_server/app.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        fetch('https://rd-api-1fd95-default-rtdb.europe-west1.firebasedatabase.app/todos.json', {
+            method: "POST",
+            body: JSON.stringify({
+                title, 
+                completed: false})
+        }).then(respons => {
+            return respons.json();
+        }).then(() => {
+            input.value = '';
+            location.reload();
```
Comment (RomanIlkiv) [inline 32_communication_server/app.js:50]: better to trigger that function `getAllTodos()` than reload the page

File: 32_communication_server/app.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+// Delete - видалити; метод DELETE - видалення; _________________________________
+function deleteTodo(id) {
+    fetch(`https://rd-api-1fd95-default-rtdb.europe-west1.firebasedatabase.app/todos/${id}.json`, {
+        method: "DELETE"})
+    .then(respons => {
+        return respons.json();
+    })
+    .then((data) => {
+      if (data === data) {
```
Comment (RomanIlkiv) [inline 32_communication_server/app.js:63]: what this if mean?
it will be always true

**Review by RomanIlkiv — APPROVED** [review]

File: 32_communication_server/index.html (hunk 1 above)
Comment (BovkunLeonid) [reply 32_communication_server/index.html:15]: Ok

File: 32_communication_server/app.js (hunk 2 above)
Comment (BovkunLeonid) [reply 32_communication_server/app.js:36]: Ок

**Pull request merged**